core/src/main/java/com/sequenceiq/cloudbreak/repository/StackRepository.java

```diff
@@ -59,6 +59,10 @@
             + "WHERE s.environmentCrn= :environmentCrn AND s.type = :type AND s.terminated=null")
     List<StackStatusView> findByEnvironmentCrnAndStackType(@Param("environmentCrn") String environmentCrn, @Param("type") StackType type);
 
+    @Query("SELECT s FROM Stack s WHERE s.environmentCrn= :environmentCrn AND s.type = :type AND s.workspace.id= :workspaceId AND s.terminated=null")
```

Comment:
I checked the code locally and the StackService method that using this repo call is used nowhere. Can we delete this repo call please?
I think it could be expensive, we should avoid fetching entire stacks from the database where it is possible.